pubtools/_quay/operator_pusher.py

```diff
@@ -453,25 +453,73 @@ def build_index_images(self):
 
         """
         iib_results = {}
+        repos_opted_in = {}
+        items_opted_in = {}
+        failed_items = {}
 
         for version, items in sorted(self.version_items_mapping.items()):
-            is_hotfix = any([item.metadata.get("com.redhat.hotfix") for item in items])
+            non_fbc_items = []
+            osev_tuple = tuple([int(x) for x in version.replace("v", "").split(".")])
+            for item in items:
+                for repo in item.metadata["tags"]:
+                    if repo not in repos_opted_in:
+                        repos_opted_in[repo] = pyxis_get_repo_metadata(
```

Comment:
It seems that the only purpose of this metadata is to throw an error if it has an unexpected value.

Reply:
Just curios question: There is no pyxis API to request get metadata in bulk?

Reply:
> Just curios question: There is no pyxis API to request get metadata in bulk?

I guess there is, but that would require now entry point in pubtools-pyxis and all together wouldn't be reviewed in time. Calling repos one by one isn't going to cause significant performance degradation